agents/config/agent_config.go

```diff
@@ -17,6 +18,8 @@ import (
 type AgentConfig struct {
 	// DBConfig is the database configuration.
 	DBConfig DBConfig `yaml:"db_config"`
+	// ScribeConfig is the scribe configuration.
+	ScribeConfig ScribeConfig `yaml:"scribe_config"`
```

Comment:
Not something to address in this pr, but as @aureliusbtc pointed out I think the entire process of setting up all these configs is extremely confusing when not using an embedded scribe. tbh, I think I pushed a bit too much for flexibility over canonicality when speccing out embedded vs remote scribe. 

Anyway, we can address this in #1274, I just wanted to note this here